packages/dataviews/src/types/field-api.ts

```diff
@@ -320,6 +325,21 @@ export type NormalizedField< Item > = Omit< Field< Item >, 'Edit' > & {
  */
 export type Fields< Item > = Field< Item >[];
 
+export type FieldValidity = {
```

Comment:
Why limit it to "required", "elements" and "custom". I believe we need to absorb more later (min, max...)... So might be better to just keep it open ended and fields can even have their own custom keys.